avocado/utils/service.py

```diff
@@ -661,19 +645,6 @@ class _SystemdServiceManager(_GenericServiceManager):
     Concrete class that implements the SystemdServiceManager
     """
 
-    def __init__(self, service_command_generator, service_result_parser,
-                 run=process.run):
-        """
-        Create the GenericServiceManager for systemd services.
-
-        :param service_command_generator:
-        :type service_command_generator: _ServiceCommandGenerator
-        :param run: function to call to run the commands, default process.run
-        :type run: function
-        """
-        super(_SystemdServiceManager, self).__init__(service_command_generator,
-                                                     service_result_parser, run)
-
```

Comment:
yes as I understand this, parent `__init__` method is already called when parent class is initiated on line 705.

Reply:
Basically this statement adds nothing. When the method is not overridden, parent's one is called. Here we override it only to call parent, so it basically does nothing.